python/py-graveyard/Portfile

```diff
@@ -128,8 +128,8 @@ py-distutils-extra      2.39_1      33
 py-django               1.11.11_1   26 33 34
 py-django-debug-toolbar 0.9.4_1     26
 py-django-extensions    1.1.1_1     26
-py-django-htmlmin       1.2_1       26
-py-django-nose          0.7.0_1     26
+py-django-htmlmin       0.10.0_1    26 34
+py-django-nose          1.2_1       26 34
```

Comment:
You can remove `26` from both. They were added here in 2016 and can go now.

Reply:
sure - one could do that, but the argument that subports were added to the graveyard a long time ago is probably valid for most things listed there. Of note, the version numbers for these ports were, as far as I can tell, inverted in the original commit 3fe5419ece35940c483931a963a252b633098fbb; I corrected that here. So any reason that I might be missing for why one should remove these specific py26 subport from py-graveyard, while all others stay?

Reply:
It's not that the others should stay. They should go as well after one year. The problem is that it's not entirely straightforward to figure out when the individual ports were added and when they could be removed. For perl it's easy because we add them all at once and we can also remove them all at once. For python it's getting more and more annoying as it takes forever to run portindex on that file. We should remove all the subports older than 1 year. We probably need to checkout a file from one year ago, compare the ports and remove those that have been in the file back then. And repeat that process on regular basis.

Reply:
okay, I understand now. It was not clear to me that the idea is to remove ports from py-graveyard. That does make sense though and a one year period seems reasonable, as this is the same time as when ports are removed after making them obsolete. 

If the consensus is that this should happen, then this should be documented in the py-graveyard Portfile as I couldn't find any information about this intention, but admittedly I only looked quickly. I will open a ticket for this in the near future and can take a look at some semi-automated way of doing this; in the meantime, feel free to merge this PR.

Reply:
@mf2k: I never claimed that this should be the blocker for merging it, I merely pointed out that it would be nice to do it, at some point. What cought my attention was precisely the mixup in version numbering.